frontend/src/Components/Teams/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+import { Layout, Card, Collapse, Typography } from "antd";
+import "./index.css";
+
+import { Api } from "../../Api";
+import Team from "../Team";
+
+const Teams = () => {
+  const { Content } = Layout;
+  const { Panel } = Collapse;
+  const { Title } = Typography;
+  const [teams, setTeams] = useState(null);
+  const [team_panels, setTeam_panels] = useState(null);
+
+  useEffect(() => {
+    Api.get("/teams/")
+      .then((result) => {
+        setTeams(result.data);
+      })
+      .catch((err) => console.log(err));
+  }, []);
+
+  useEffect(() => {
+    if (teams) {
```

Comment:
This whole logic can be done in the same `then` function as `setTeams` is being called